### ISSUE TITLE
Error on parallelize plugin

### ISSUE THREAD

**Issue by lelea2** [tildeio/ember-element-helper#32]
I got this error `[broccoli-persistent-filter:Babel > [Babel: ember-element-helper]: Babel: ember-element-helper] was configured to `throwUnlessParallelizable` and was unable to parallelize a plugin` while trying to build with the set. up

```
  'ember-cli-babel': { 
      includeExternalHelpers: true, 
      throwUnlessParallelizable: true, 
    }, 
 
```

**Comment by mrloop** [comment]
I'm experiencing the same issue, created a example project demonstrating it https://github.com/mrloop/ember-element-helper-parallelizable

**Comment by mrloop** [comment]
Ah there's already a PR #33 , thanks @lelea2 :+1:

**Comment by rwjblue** [comment]
Fixed by #33
